k8s/dialer.go

```diff
@@ -102,6 +102,7 @@ func (c *contextDialer) startDialerPod(ctx context.Context) (err error) {
 	if err != nil {
 		return
 	}
+	c.restConf.WarningHandler = restclient.NoWarnings{}
```

Comment:
why is this needed ?

Reply:
that's not exactly needed, it should be rather used depending on `--verbose -v` flags
so user is not scared in non verbose mode

Reply:
I just don't want to scare users.

Reply:
I see, fine for me.